lib/plan_picker_web/templates/class_management/index.html.eex

```diff
@@ -0,0 +1,49 @@
+<h1 class="title"> Classes in <%= @enrollment.name %> </h1>
+<%= link "Back to enrollment", to: Routes.enrollment_path(@conn, :show, @enrollment.id) %>
+
+<div class="columns">
+    <div class="column is-one-quarter">
+        <%= if length(@enrollment.users) > 0 do %>
+            <%= for user <- @enrollment.users do %>
+                <div class="block">
+                    <%= user.index_no %>, <%= user.name %> <%= user.last_name %>
+                </div>
+        <% end %>
+        <% else %>
+            <div class="block">
+                There are no users assigned to this enrollment.
+            </div>
+        <% end %>
```

Comment:
This can be a partial template, it's used multiple times.

Reply:
Done.